test/e2e/utils_test.go

```diff
@@ -799,6 +799,20 @@ func cleanupCRP(name string) {
 	Eventually(removedActual, workloadEventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to remove CRP %s", name)
 }
 
+// cleanupCRPDB deletes the CRPDB and waits until the resource is not found.
+func cleanupCRPDB(name string) {
```

Comment:
We already have this function in place, https://github.com/kubefleet-dev/kubefleet/blob/main/test/e2e/utils_test.go#L964-L973